packages/core/sdk/src/auth/internal/CoreStitchAuth.ts

```diff
@@ -701,4 +898,39 @@ export default abstract class CoreStitchAuth<TStitchUser extends CoreStitchUser>
       this.accessTokenRefresher.stop();
     }
   }
+
+  private clearUserAuth(userId: string) {
+    const unclearedAuthInfo = this.allUsersAuthInfo.get(userId);
+    if (unclearedAuthInfo === undefined) {
+      // this doesn't necessarily mean there's an error. we could be in a 
+      // provisional state where the profile request failed and we're just
+      // trying to log out the active user.
+      if (this.activeUserAuthInfo.userId !== userId) {
+        // only throw if this ID is not the active user either
+        throw new StitchClientError(StitchClientErrorCode.UserNotFound);  
+      }
+    }
+
```

Comment:
This kind of seems like weird behaviour / should change the name of this function. When I hear clearUserAuth I assume that this method would delete the user entirely from the list of users, not just log them out

Reply:
per the other PR, I think `clearUserAuthTokens` is the most appropriate